This removes an invalid assumption that results in both BLARGG_LITTLE_ENDIAN
and BLARGG_BIG_ENDIAN being defined on little endian musl, as we're already
covered by __LITTLE_ENDIAN__/__BIG_ENDIAN__ (which is specified as a part
of the ELFv2 ABI itself to be always present). Glibc was not broken because
it has its own check before that.

--- gme/blargg_endian.h
+++ gme/blargg_endian.h
@@ -36,7 +36,7 @@
 #endif
 
 #if defined (MSB_FIRST)     || defined (__BIG_ENDIAN__) || defined (WORDS_BIGENDIAN) || \
-	defined (__sparc__)     ||  BLARGG_CPU_POWERPC || \
+	defined (__sparc__) || \
 	(defined (BIG_ENDIAN) && BIG_ENDIAN+0 != 4321)
 	#define BLARGG_BIG_ENDIAN 1
 #elif !defined (__mips__)
